tests/AulaTests.hs

```diff
@@ -45,11 +46,24 @@ import Logger (LogLevel(..), nullLog)
 import Persistent (mkMetaInfo)
 
 
--- | 'runIO" with better errors.
+-- | 'runIO' with better errors.
```

Comment:
haddocks allows `'runIO''`, I think, but what I really want to say here is that `runIO'` is like `runIO`, but with better errors.  So I'm actually referring to `runIO`, not `runIO'`.

i'm confused now.  (-: